lib/backpex/fields/select.ex

```diff
@@ -1,11 +1,24 @@
 defmodule Backpex.Fields.Select do
+  @config_schema [
+    options: [
+      doc: "List of options or function that receives the assigns.",
+      type: {:or, [{:list, :any}, {:fun, 1}]},
+      required: true
+    ],
+    prompt: [
+      doc: "The text to be displayed when no option is selected or function that receives the assigns.",
+      type: {:or, [:string, {:fun, 1}]}
+    ]
+  ]
```

Comment:
Is `debounce` and `throttle` missing here? It is used in `render_form/1` @pehbehbeh